src/components/value.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+import * as AppPropTypes from "../app-prop-types"
+
+
+const Value = React.createClass({
+  propTypes: {
+    children: AppPropTypes.value.isRequired,
+  },
+  render() {
+    const {children} = this.props
+    return <samp>{String(children)}</samp>
+  }
+})
+
+
+export default Value
```

Comment:
Thanks for the references.

What I still don't understand is why we have to use `String` in the first place. I tried locally to replace <Value> by <samp>, and it seems to render strictly the same way. Am I missing something ?